linkml/generators/common/ifabsent_processor.py

```diff
@@ -31,6 +38,34 @@
     Uriorcurie,
 )
 
+TYPES_TYPE: TypeAlias = Union[
+    Type[Boolean],
+    Type[Curie],
+    Type[Date],
+    Type[DateOrDatetime],
+    Type[Datetime],
+    Type[Decimal],
+    Type[Double],
+    Type[Float],
+    Type[Integer],
+    Type[Jsonpath],
+    Type[Jsonpointer],
+    Type[Ncname],
+    Type[Nodeidentifier],
+    Type[Objectidentifier],
+    Type[Sparqlpath],
+    Type[String],
+    Type[Time],
+    Type[Uri],
+    Type[Uriorcurie],
+]
+
+TYPES = [
+    t
+    for t in types.__dict__.values()
+    if isinstance(t, type) and t.__module__ == types.__name__ and hasattr(t, "type_name")
+]
```

Comment:
i also think that this kind of thing would be great to have in a `linkml-meta` repository which just has the metamodel and metamodel accessories in it. couldn't find a way to get "all the base linkml types" anywhere

Reply:
testing resolving this comment @sneakers-the-rat - just so I can see if merging is a possibility